<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-380UJ2J" failures="2" tests="2" name="Default test(failed)" time="0.037" errors="0" timestamp="2023-01-02T00:33:56 IST">
  <testcase classname="testNG_xml_study.Neostox1" name="c" time="0.006">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG_xml_study.Neostox1.c(Neostox1.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- c -->
  <testcase classname="testNG_xml_study.Neostox1" name="b" time="0.0">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG_xml_study.Neostox1.b(Neostox1.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- b -->
  <testcase classname="testNG_xml_study.Neostox1" name="d" time="0.0">
    <ignored/>
  </testcase> <!-- d -->
  <testcase classname="testNG_xml_study.Neostox1" name="a" time="0.0">
    <ignored/>
  </testcase> <!-- a -->
</testsuite> <!-- Default test(failed) -->
